src/output.rs

```diff
@@ -82,6 +89,10 @@ impl OutputType {
                             p.arg("--quit-if-one-screen");
                         }
 
+                        if !line_wrap {
+                            p.arg("-S");
```

Comment:
Minor thing, but can we please use the long option `--chop-long-lines`.

Reply:
Sure, seems better.